doc/config_rsmtool.rst

```diff
@@ -207,6 +207,17 @@ If set to ``true``, certain evaluations (confusion matrices, score distributions
     All evaluation metrics (e.g., kappa and pearson correlation) are automatically computed for *both* scaled and raw scores.
 
 
+.. _use_truncations:
+
+use_truncations *(Optional)*
```

Comment:
`use_truncation_thresholds`  - this is to avoid giving impression that when set to False you won't use truncation at all.